Libs/Core/ctkUtils.h

```diff
@@ -24,12 +24,28 @@
 // Qt includes
 #include <QStringList>
 #include <QDateTime>
+#include <QDebug>
 
 // STD includes
 #include <vector>
 
 #include "ctkCoreExport.h"
 
+/// This macro can be used instead of Q_ASSERT to warn developers
+/// when some assumption fails. CTK_SOFT_ASSERT behavior differs
+/// in two key aspectsit only logs a warning (instead of
+/// terminating the application) and the message is logged
+/// in both release and debug mode builds (instead of ignoring
+/// the condition in release mode).
```

Comment:
```
[...]
/// in two key aspects: (1) it only logs a warning (instead of
/// terminating the application) and (2) the message is logged
/// in both release and debug mode builds (instead of ignoring
/// the condition in release mode).
```

Reply:
Thanks, fixed.